ansible/roles/postgresql/templates/postgresql.conf.j2

```diff
@@ -201,7 +200,7 @@ vacuum_cost_delay = 20ms        # 0-100 milliseconds
 
 # These settings are ignored on a standby server
 
-#max_wal_senders = 0        # max number of walsender processes
+max_wal_senders = 3        # max number of walsender processes
```

Comment:
`max_wal_senders` is necessary for `pg_basebackup`, and therefore our backup scripts, so its potentially useful even in deployments without a standby, but more generally, neither setting should affect the normal running of a postgres server. The `hot_standby` is explicitly ignored if the server isn't in recovery mode and increasing the `max_wal_senders` didn't seem to have a downside on a running machine